.circleci/config.yml

```diff
@@ -42,10 +44,15 @@ jobs:
       - run:
           name: Run TestStart Golang tests
           command: |
-            go test -v -covermode=atomic -coverprofile=profile.out \
-            -run TestStart && \
-            /bin/bash <(curl -s https://codecov.io/bash) -f profile.out
-            if [[ $? -ne 0 ]]; then cat db_test_err.log; exit $?; fi
+            go test -v -covermode=atomic -coverprofile=profile.out -run TestStart
+            if [[ $? -ne 0 ]]; then
```

Comment:
This is the fix for the false negatives

Reply:
this was returning 0 for all calls. You can only evaluate $? one time in the if statement since we are checking other values $? is reset to the evaluation itself so exiting on $? after $? -ne 1 is invalid